README.md

```
@@ -1 +1,431 @@
+
+This starts three containers:
+
+| Service    | Container  | Port          |
+|------------|------------|---------------|
+| API + gRPC | `grn-app`  | `3000`, `50051` |
+| PostgreSQL | `grn-db`   | `5433` (host) → `5432` (container) |
+| Redis      | `grn-redis`| `6379`        |
+
+Once all containers are healthy, verify the app is running:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)